presto-kafka/src/main/java/com/facebook/presto/kafka/KafkaConnectorConfig.java

```diff
@@ -69,6 +63,38 @@
      */
     private boolean hideInternalColumns = true;
 
+    /**
+     * The Zookeeper URI.
+     * Example: hostAddress:2181
+     */
+    private String kafkaZookeeperUri = "sdzookeeper.smf1.twitter.com:2181";
+
+    /**
+     * The Zookeeper path for Kafka services.
+     * Example: /twitter/service/kafka/prod/tfe
+     */
+    private String kafkaServicePath = "/twitter/service/kafka/prod/tfe";
+
+    /**
+     * Maximum number of records per poll()
+     */
+    private int maxPollRecords = 500;
+
+    /**
+     * Maximum number of bytes from one partition per poll()
+     */
+    private int maxPartitionFetchBytes = 1048576;
```

Comment:
do we need to tune these numbers in the future? If so, shift them to a config file.

Reply:
Yes, we might need to. It is configurable.